services/eventservice/src/tests.rs

```diff
@@ -0,0 +1,268 @@
+use crate::types::{Category, Event};
+
+use super::*;
+
+fn test_categories() -> (Category, Category, Category, Category) {
+    let cat1 = "first".to_string();
+    let cat2 = "second".to_string();
+    let cat3 = "third".to_string();
+    let cat4 = "fourth".to_string();
+    (cat1, cat2, cat3, cat4)
+}
+
+fn test_events(cat1 : Option<Category>, cat2 : Option<Category>,cat3 : Option<Category>) -> (Event,Event,Event) {
```

Comment:
Please, use test fixtures to move all unnecessary setup and cleanup from tests. This will make them cleaner and easier to understand.

Reply:
What is test fixtures?

Reply:
https://en.wikipedia.org/wiki/Test_fixture

Reply:
Rust native tests don't have setup and teardown analogue, so it will be nice to write maybe some macro, but let's move it to another task